<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClassClass</class>
 <widget class="QMainWindow" name="MyClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>941</width>
      <height>691</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add class</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="listTab">
     <attribute name="title">
      <string>List class</string>
     </attribute>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchLineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="listTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>911</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="addClassTab">
     <attribute name="title">
      <string>Add class</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="addCourseTab">
     <attribute name="title">
      <string>Add course</string>
     </attribute>
     <widget class="QGroupBox" name="classInsertgGroupBox">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>321</width>
        <height>641</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add skills</string>
      </property>
      <property name="title">
       <string>Class inserted :</string>
      </property>
     </widget>
     <widget class="QWidget" name="step1Widget" native="true">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <widget class="QLabel" name="step1Label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>291</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Step 1 :Course name (e.g: Ielts, Toefl,...)</string>
       </property>
      </widget>
      <widget class="QLabel" name="setCourseLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Set course name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="courseNameLineEdit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>0</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myclass.qrc">
         <normaloff>:/MyClass/Resources/saveButtton.jpg</normaloff>:/MyClass/Resources/saveButtton.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="step2Widget" native="true">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>381</width>
        <height>161</height>
       </rect>
      </property>
      <widget class="QLabel" name="step2Label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Step 2: Add skills</string>
       </property>
      </widget>
      <widget class="QListWidget" name="leftWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>121</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="rightWidget">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>121</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="left2RightButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="right2LeftButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="all2RightButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>all &gt;&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="all2LeftButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;&lt; all</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>0</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save skills&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myclass.qrc">
         <normaloff>:/MyClass/Resources/saveButtton.jpg</normaloff>:/MyClass/Resources/saveButtton.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>294</x>
         <y>0</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add Skills&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myclass.qrc">
         <normaloff>:/MyClass/Resources/add-icon.png</normaloff>:/MyClass/Resources/add-icon.png</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="step3Widget" native="true">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>381</width>
        <height>371</height>
       </rect>
      </property>
      <widget class="QLabel" name="step2Label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Step 3: Add materials</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton3">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>0</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myclass.qrc">
         <normaloff>:/MyClass/Resources/saveButtton.jpg</normaloff>:/MyClass/Resources/saveButtton.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClass">
    <property name="title">
     <string>Class</string>
    </property>
    <addaction name="actionList"/>
    <addaction name="actionAdd_class"/>
   </widget>
   <addaction name="menuClass"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="separator"/>
   <addaction name="actionCreateNew"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionAdd_class">
   <property name="text">
    <string>Add class</string>
   </property>
  </action>
  <action name="actionList">
   <property name="text">
    <string>List class</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="myclass.qrc">
     <normaloff>:/MyClass/Resources/openFile.jpg</normaloff>:/MyClass/Resources/openFile.jpg</iconset>
   </property>
   <property name="text">
    <string>openFile</string>
   </property>
   <property name="toolTip">
    <string>Click file to open </string>
   </property>
  </action>
  <action name="actionCreateNew">
   <property name="icon">
    <iconset resource="myclass.qrc">
     <normaloff>:/MyClass/Resources/createNew.jpg</normaloff>:/MyClass/Resources/createNew.jpg</iconset>
   </property>
   <property name="text">
    <string>createNew</string>
   </property>
   <property name="toolTip">
    <string>Create new anything</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="myclass.qrc">
     <normaloff>:/MyClass/Resources/edit.jpg</normaloff>:/MyClass/Resources/edit.jpg</iconset>
   </property>
   <property name="text">
    <string>edit</string>
   </property>
   <property name="toolTip">
    <string>edit something</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="myclass.qrc">
     <normaloff>:/MyClass/Resources/Delete_icon.png</normaloff>:/MyClass/Resources/Delete_icon.png</iconset>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="myclass.qrc">
     <normaloff>:/MyClass/Resources/save_icon.png</normaloff>:/MyClass/Resources/save_icon.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myclass.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpenFile</sender>
   <signal>triggered()</signal>
   <receiver>MyClassClass</receiver>
   <slot>openFileAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCreateNew</sender>
   <signal>triggered()</signal>
   <receiver>MyClassClass</receiver>
   <slot>createAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit</sender>
   <signal>triggered()</signal>
   <receiver>MyClassClass</receiver>
   <slot>editAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete</sender>
   <signal>triggered()</signal>
   <receiver>MyClassClass</receiver>
   <slot>deleteAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MyClassClass</receiver>
   <slot>saveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>MyClassClass</receiver>
   <slot>step1SaveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton2</sender>
   <signal>clicked()</signal>
   <receiver>MyClassClass</receiver>
   <slot>step2SaveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>764</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton3</sender>
   <signal>clicked()</signal>
   <receiver>MyClassClass</receiver>
   <slot>step3SaveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>764</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>left2RightButton</sender>
   <signal>clicked()</signal>
   <receiver>MyClassClass</receiver>
   <slot>left2RightAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>768</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>right2LeftButton</sender>
   <signal>clicked()</signal>
   <receiver>MyClassClass</receiver>
   <slot>right2LeftAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>784</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>all2RightButton</sender>
   <signal>clicked()</signal>
   <receiver>MyClassClass</receiver>
   <slot>all2RightAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>758</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>all2LeftButton</sender>
   <signal>clicked()</signal>
   <receiver>MyClassClass</receiver>
   <slot>all2LeftAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>775</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leftWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MyClassClass</receiver>
   <slot>left2RightClickAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>762</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MyClassClass</receiver>
   <slot>right2LeftClickAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>773</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFileAction()</slot>
  <slot>saveAction()</slot>
  <slot>editAction()</slot>
  <slot>deleteAction()</slot>
  <slot>createAction()</slot>
  <slot>step1SaveAction()</slot>
  <slot>step2SaveAction()</slot>
  <slot>step3SaveAction()</slot>
  <slot>left2RightAction()</slot>
  <slot>right2LeftAction()</slot>
  <slot>all2RightAction()</slot>
  <slot>all2LeftAction()</slot>
  <slot>left2RightClickAction()</slot>
  <slot>right2LeftClickAction()</slot>
 </slots>
</ui>
